src/apphosting/config.ts

```diff
@@ -67,12 +84,13 @@ export function discoverBackendRoot(cwd: string): string | null {
 /**
  * Lists absolute paths for `apphosting.*.yaml` configs at backend root
  */
-export function discoverConfigsAtBackendRoot(cwd: string): string[] {
-  const backendRoot = discoverBackendRoot(cwd);
+export function discoverConfigsAtBackendRoot(cwd: string, rootDir?: string): string[] {
```

Comment:
This function might look better if you pass in a single directory path and resolve the `cwd` vs `rootDir` beforehand, eg

```
backendRoot = backendRoot ? join(cwd, backendRoot) : cwd
discoverConfigs(backendRoot)
```